proxysql/Dockerfile.k8s

```diff
@@ -23,7 +23,7 @@ All the while getting the unlimited freedom that comes with a GPL license."
 LABEL org.opencontainers.image.license="GPL+"
 LABEL org.opencontainers.image.authors="info@percona.com"
 
-ENV PXC_VERSION 8.0.21-12.1
+ENV PXC_VERSION 8.0.22-13.1
 ENV PROXYSQL_VERSION 2.0.15-1.1
```

Comment:
maybe we need to update it too...

Reply:
Verified with Evgeniy and it seems ok to bump the version now, also added `USE_STDIN_FOR_CREDENTIALS=1` to proxysql-admin.cnf so please check it